ee/frontend/mobile-replay/transformer/screen-chrome.ts

```diff
@@ -0,0 +1,78 @@
+import { NodeType, serializedNodeWithId, wireframeDiv, wireframeStatusBar } from '../mobile.types'
+import { makeDivElement, STATUS_BAR_ID } from './transformers'
+import { ConversionContext, ConversionResult } from './types'
+import { asStyleString, makeStylesString, TOP_OF_STACK } from './wireframeStyle'
+
+function spacerDiv(idSequence: Generator<number>): serializedNodeWithId {
+    const spacerId = idSequence.next().value
+    return {
+        type: NodeType.Element,
+        tagName: 'div',
+        attributes: {
+            style: 'width: 5px;',
+            'data-rrweb-id': spacerId,
+        },
+        id: spacerId,
+        childNodes: [],
+    }
+}
+
+export function makeNavigationBar(
+    wireframe: wireframeDiv,
+    _children: serializedNodeWithId[],
+    context: ConversionContext
+): ConversionResult<serializedNodeWithId> | null {
+    return makeDivElement(wireframe, _children, {
+        ...context,
+        styleOverride: { ...context.styleOverride, 'z-index': TOP_OF_STACK },
+    })
+}
```

Comment:
file was renamed from status-bar

adds this method for navigation bar to set an explicit z-index